app/views/audit_mailer/summary.text.erb

```diff
@@ -0,0 +1,16 @@
+<%= _("Foreman Audit summary") %>
+=================================
+<% if @audits.size > 0 %>
+  <%= _('Displaying %{num_audits} audits of %{total_audits} audits') % {:num_audits => @limit, :total_audits => @count} %>
+  <% @audits.each do |audit| %>
+      ______________________________________________________________
+      |  <%= audit.username.gsub(_('User'), '') %>                 |
+      |  ( <%= audit.remote_address %> ")"                         |
+      |  <%= audit_action_name audit%> <%= audited_type audit %>:  |
+      |  <%= audit_title(audit) %>                                 |
```

Comment:
Couldn't the audit title make the box boundries not line up? It looks nice, but because of that I might avoid it?